src/Middleware/UnauthorizedHandler/RedirectHandler.php

```diff
@@ -44,6 +44,7 @@ class RedirectHandler implements HandlerInterface
         'url' => '/login',
         'queryParam' => 'redirect',
         'statusCode' => 302,
+        'allowedRedirectExtensions' => false,
```

Comment:
We should add this new option to the docblock.

Reply:
Done
I switched the default to true, as that is more logical.